### PR TITLE
Deepgram STT support

### PR DESCRIPTION
**Pull request by Wrsanches**
- Add Deepgram STT support
- Remove `dist/` folder when a new build is created
- Suppress unnecessary warnings
- Add NPM badge to README

### CODE REVIEW

File: rollup.config.mjs [hunk 1]
```diff
@@ -44,6 +46,14 @@ export default [
           { src: 'package.json', dest: 'dist' },
         ],
       }),
+      del({ targets: 'dist/*' }),
```
Comment (gfpacheco) [inline rollup.config.mjs:49]: suggestion: move this up first in the plugins array, even it doesn't need to, just for the sake of readability

File: rollup.config.mjs [hunk 2]
```diff
@@ -44,6 +46,14 @@ export default [
...
     ],
+    onwarn: (warning, next) => {
+      if (warning.code === 'CIRCULAR_DEPENDENCY') return;
+      if (warning.code === 'THIS_IS_UNDEFINED') return;
+      if (warning.code === 'PLUGIN_WARNING') return;
```
Comment (gfpacheco) [inline rollup.config.mjs:54]: suggestion: ignore third party code warnings
```suggestion
      if (
        (warning.id && /node_modules/.test(warning.id)) ||
        (warning.ids && warning.ids.every((id) => /node_modules/.test(id)))
      ) {
        return;
      }
```

File: src/plugins/stt/deepgram/DeepgramSpeechRecognition.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      const deepgram = createClient(apiKey);
+      this.live = deepgram.listen.live({
+        model: 'nova-2',
+        language: 'en-US',
+        ...deepgramOptions,
+      });
+
+      this.live.on(LiveTranscriptionEvents.Open, () => {
+        console.log('started speech recognition');
```
Comment (gfpacheco) [inline src/plugins/stt/deepgram/DeepgramSpeechRecognition.ts:24]: suggestion: remove all logs from our sdk (until we have a proper logging that is only enabled if the user wants to)

**Review by gfpacheco — APPROVED** [review]

File: rollup.config.mjs (hunk 1 above)
Comment (Wrsanches) [reply rollup.config.mjs]: Done: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/7/commits/f332ea1976b8b8928986085970af107313ded53e

File: rollup.config.mjs (hunk 2 above)
Comment (Wrsanches) [reply rollup.config.mjs]: Done: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/7/commits/f332ea1976b8b8928986085970af107313ded53e

File: src/plugins/stt/deepgram/DeepgramSpeechRecognition.ts (hunk 3 above)
Comment (Wrsanches) [reply src/plugins/stt/deepgram/DeepgramSpeechRecognition.ts]: Done: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/7/commits/f332ea1976b8b8928986085970af107313ded53e

**Pull request merged**